### PR TITLE
Add split command to photos example

### PR DESCRIPTION
**Pull request by mberhault**
splits all three tables.

---
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/examples-go/90)

### CODE REVIEW

File: photos/main.go [hunk 1]
```diff
@@ -168,10 +180,67 @@ func runDrop(c *cobra.Command, args []string) error {
...
+
+	if err := initSchema(db); err != nil {
+		log.Fatal(err)
+	}
+	ctx.DB = db
+
+	log.Printf("splitting table \"users\"")
+	for count := 0; count < numSplits; {
+		// Use the userID generation logic.
+		userID := 1 + int(rand.ExpFloat64()/rate)
```
Comment (a-robinson) [inline photos/main.go]: Might be worth pulling this out into a function

Comment (mberhault) [inline photos/main.go]: yeah. I'm still poking at it. the photos example has other problems running against a growing cluster (or rather, the cluster has problems), so I can't even tell how effective the pre-splits are.

File: photos/main.go [hunk 2]
```diff
@@ -168,10 +180,67 @@ func runDrop(c *cobra.Command, args []string) error {
...
+		log.Fatal(err)
+	}
+	defer func() { _ = db.Close() }()
+
+	if err := initSchema(db); err != nil {
+		log.Fatal(err)
+	}
+	ctx.DB = db
+
+	log.Printf("splitting table \"users\"")
```
Comment (petermattis) [inline photos/main.go]: Using a back-quoted string can remove the need for escaping:

```
log.Printf(`splitting table "users"`)
```

Comment (mberhault) [inline photos/main.go]: done

File: photos/main.go (hunk 1 above)
Comment (mberhault) [inline photos/main.go]: done.

**Pull request merged**